### PR TITLE
modify default access for opendevstack users to bitbucket project and…

### LINKED ISSUE #211: modify default access for opendevstack users to bitbucket project and repository
When a bitbucket **project** is created, the following default permission should be granted:
- Every member of the opendevstack-users group (as configured via property `idmanager.group.opendevstack-users`)  should have **read access** at project level

When a bitbucket **repository** is created, the following default permission should be granted:
- Every member of the opendevstack-users group (as configured via property idmanager.group.opendevstack-users) should have **write access** to the generated repository

Any thought about this?

### PR DESCRIPTION
**Pull request by stefanlack**
… repository 

fixes #211

### CODE REVIEW

**Comment by stefanlack** [comment]
@rattermeyer : thank you for your review. 
@michaelsauter : any thoughts from your site regarding this PR?

File: src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java [hunk 1]
```diff
@@ -279,8 +289,10 @@ protected RepositoryData callCreateRepoApi(String projectKey, Repository repo)
                   + " - no response from endpoint, please check logs",
               repo.getName(), projectKey));
     }
-    setRepositoryPermissions(data, projectKey, ID_GROUPS, repo.getUserGroup());
-    setRepositoryPermissions(data, projectKey, ID_USERS, technicalUser);
+    setRepositoryAdminPermissions(data, projectKey, ID_GROUPS, repo.getUserGroup());
+    setRepositoryAdminPermissions(data, projectKey, ID_USERS, technicalUser);
+    setRepositoryPermissions(
```
Comment (clemensutschig) [inline src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java]: @stefanlack  - that one I am NOT ok with ... that means everyone has write access .. (and we removed that long time ago ..) why is this needed...?

Comment (stefanlack) [reply src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java]: @clemensutschig because it is our philosophy for OpenDevStack@OPITZ and was requested by @rattermeyer .
But I can see your thoughts about that. We also had some internal discussions.

Suggested Solution: 
- Implementing an property `provision.scm.grantglobalwritetouser` (default value=false). 

By setting to true, the  line `setRepositoryPermissions(data, projectKey, ID_GROUPS, openDevStackUsersGroupName, REPOSITORY_PERMISSIONS.REPO_WRITE);` is executed, otherwise it is not executed


>

Comment (stefanlack) [reply src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java]: Is this ok for you?

Comment (clemensutschig) [reply src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java]: @stefanlack .. there was a rule in the codebase.. if no special perm set.. then its ok... in case of special perm set.. no addtl write rules

**Comment by rattermeyer** [comment]
@clemensutschig can you please check, if you are fine with the changes?
